MAUI/DataGrid/columns.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: Columns in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Columns support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control, its elements and more.

```

Comment:
description: Learn all about Columns support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control, its elements, and more here.

Reply:
Modified.